app/src/main/assets/html/error/css/styles.css

```diff
@@ -0,0 +1,173 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+/* open-sans-regular - latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Open Sans Regular'), local('OpenSans-Regular'), url('../fonts/open-sans/open-sans-v15-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */ url('../fonts/open-sans/open-sans-v15-latin-regular.woff') format('woff');
+  /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+/* open-sans-700 - latin */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Open Sans Bold'), local('OpenSans-Bold'), url('../fonts/open-sans/open-sans-v15-latin-700.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */ url('../fonts/open-sans/open-sans-v15-latin-700.woff') format('woff');
+  /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+* {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 24px;
+  line-height: 1.2em;
+}
+html,
+body {
+  display: grid;
+  grid-template-rows: auto;
+  justify-items: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background: #f4f4f4;
+}
+a {
+  color: black;
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: underline;
+  text-decoration-style: dotted;
+}
+hr {
+  margin: 30px 0;
+}
+h1 {
+  font-size: 60px;
+  font-weight: bold;
+  text-align: center;
```

Comment:
personally, I wouldn't center these error messages, for l10n purposes

Reply:
So your suggesting that we use default text alignment in general due to localization?  I was just following styles as per designs, but that makes sense to me.  I thought the only thing to watch out for was allowing for right-left languages, didn't know about centering.

Reply:
yes, that is correct